content/en/users/getting-started/cli/_index.md

```diff
@@ -1,11 +1,444 @@
+the following commands:
+
+<!-- markdownlint-disable line-length -->
+```shell
+$ wget https://raw.githubusercontent.com/tdviet/python-requests-bundle-certs/main/scripts/install_certs.sh
+$ bash install_certs.sh
+```
+<!-- markdownlint-enable line-length -->
+
+{{% alert title="Note" color="info" %}} The above script does not work on all
+Linux distributions. Change _python_ to _python3_ in the script if needed,
+see the [README](https://github.com/tdviet/python-requests-bundle-certs#usage)
+for more details, or follow the
+[official instructions](https://github.com/tdviet/python-requests-bundle-certs/blob/main/docs/Install_certificates.md)
+for installing EGI Core Trust Anchor certificates in production environments.
+{{% /alert %}}
+
+### Using via Docker container
+
+The FedCloud client can also be used without installation, by running it in a
+Docker container. In this case, the EGI Core Trust Anchor certificates are
+preinstalled.
+
+{{< tabpanex >}}
+{{< tabx header="Linux " >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/engine/install/#server),
+then run the following commands:
+
+```shell
+$ sudo docker pull tdviet/fedcloudclient
+$ sudo docker run -it  tdviet/fedcloudclient bash
+```
+
+{{< /tabx >}}
+{{< tabx header="Mac" >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/desktop/mac/install/),
+then run the following commands:
+
+```shell
+$ sudo docker pull tdviet/fedcloudclient
+$ sudo docker run -it  tdviet/fedcloudclient bash
+```
+
+{{< /tabx >}}
+{{< tabx header="Windows" >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/desktop/windows/install/),
+then run the following commands:
+
+```shell
+> docker pull tdviet/fedcloudclient
+> docker run -it  tdviet/fedcloudclient bash
+```
+
+{{< /tabx >}}
+{{< /tabpanex >}}
+
+Once you have a shell running in the container with the FedCloud client, usage
+is the same as from [the command line](#using-from-the-command-line).
+
+### Using from the command line
+
+The FedCloud client has these subcommands:
+
+- **fedcloud token** for checking access tokens
+  (see [subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-token-commands))
+- **fedcloud endpoint** for querying the Configuration Database
+  (see [subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-endpoint-commands))
+- **fedcloud site** for manipulating site configurations
+  (see [subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-site-commands))
+- **fedcloud openstack** or **fedcloud openstack-int** for performing
+  OpenStack commands on sites
+  (see [subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-openstack-commands))
+- **fedcloud ec3** for provisioning elastic cloud compute clusters
+  (see [subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-ec3-commands))
```

Comment:
May be use things like `[token subcommands](https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-token-commands)` to help distinguish their prupose?

Reply:
Fixed